organization/pre-workshop-survey.md

```diff
@@ -0,0 +1,84 @@
+
+1. I consent to this survey *
+
+    Yes
+
+1. Are you age 18 or above? (we should not collect if under 18)* 
+
+    Yes
+    No
+
+1. In a few words, what is your most important reason for attending this workshop?
+
+    
+
+1. Below are some reasons we think people will come to Toronto Mesh workshops. Some of these reasons may overlap with the reasons you suggested above. How important were these reasons to you?
+  	
+    _Not at all important 	Slightly important 	Important 	Very important 	N/A_  
+    Improve your knowledge of networking and the internet  
+    Join an active project or group of enthusiasts  
+    Experiment with new software and tools related to your interests  
+    Network with other workshop attendees   	
+    Learn new or update existing skills  	
+
+1. Besides the course content, what importance did the following factors have on your choice to attend the workshop?
+  	Not important at all 	Slightly important 	Important 	Very important 	Extremely important
+
+    Attending an in-person/live workshop  
+    Location and timing of workshop  
+    Learning with friends/colleagues  
+    Other (please specify)  
+
+6. How did you hear about Toronto Mesh / Building the Peer-to-Peer Internet workshop? (Select all that apply)
+
+    Friend/colleague  
+    Conference/meeting/seminar  
+    Library website or flyer  
+    Funding organization or program officer  
+    Journal or publication  
+    Our website  
+    Social Media (Twitter, Facebook, etc.)  
+    Other (please specify)  
+
+1. If you experienced any strong concerns about attending this workshop, can you let us know what they were? Did we address these concerns, or were there ways we could have addressed them? w
+
+   
+---
+
+8. Tell us about your experience with the following topics:
+
+    _Current knowledge (Little or no knowledge | Some knowledge | Extensive knowledge)_	
+    _Perception about this topic (Very intimidating | Slightly intimidating | Neither intimidating nor unintimidating | Not very intimidating | Not at all intimidating | N/A or No opinion)_  
+    Unix Shell  
+    Networking  
+    Decentralized Services  
+    Mesh Networks  
+    Microcontrollers and Singleboard Computers (e.g. Raspberry Pi or Arduino)  
+
+1. How would you rate your motivation to learn about these topics?
+
+    _N/A - Not relevant to me/Not sure what this is |	Not at all motivated | Slightly motivated | Motivated | Very motivated | Extremely motivated_  
+    Unix Shell
+    Networking
+    Decentralized Services
+    Mesh Networks
+    Microcontrollers and Singleboard Computers (e.g. Raspberry Pi or Arduino)
+
+1. How would you describe your ability to do the following tasks?
+  	Can you do this?
+
+    _Yes - No - Maybe_  
+    Use terminal or command line to list files in a directory
+    Log in or 'ssh into' a server
+    Log in to router admin panel (i.e., to change network password)
+    Update software or firmware on a router
+
+---
+
+11. What is your gender? 
```

Comment:
I don't think this is necessary

Reply:
some representation and identity information is helpful, but fine to remove (as this is only one bit of a constellation)